docs/jsonapi/jsonapi-basic-configuration.md

```diff
@@ -7,17 +7,17 @@ To apply the mixin to your models and collections, use the `jsonapi` method. Eve
 
 ```typescript
 import { Collection, Model } from '@datx/core';
-import { config, jsonapi } from '@datx/jsonapi';
+import { config, jsonapiModel } from '@datx/jsonapi';
```

Comment:
What's the reasoning here to use `jsonapiModel` and collection instead of the `jsonapi` mixin? My thinking was to decrease the API surface and only have one function for everything.

Reply:
I would assume that we don't want to bundle parts we do not need (static vs runtime). For example, view is something that is not used that often but it is being part of every app that uses datx.

Reply:
What @kristian240  said :D

Reply:
@DarkoKukovec Are we okay with this then? If yes, can someone approve the PR? 😄